CHANGELOG.rst

```diff
@@ -24,7 +24,7 @@ Deprecations:
 Changes:
 ^^^^^^^^
 
-*none*
+- Added AWS SecretsManager support.
```

Comment:
Could you make the name a link to the product page or to some documentation? It could be that some AWS user runs into it and never heard of it. This could save someone's ass. :)